### PR TITLE
Heat_units.md

### PR DESCRIPTION
**Pull request by LaplacetteCM**
Heat units wiki to be reviewed.

### CODE REVIEW

File: SC_redesign/Crop_and_Soil/docs/heat_units.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+2. **Alternative method**: `heat_unit_temperature` is used instead of `air_temperature`. 
+Heat units accumulated are defined using the `heat_unit_temperature` resulting from the 
+comparison between minimum and maximum daily temperatures of crop and air:
+
+**Minimum heat unit temperature** calculates the minimum heat unit temperature on a given
+day using:
+```math
+  $$ Tmin = 
+  max(TminAir, TminCrop)     
+$$
```
Comment (morrowcj) [inline SC_redesign/Crop_and_Soil/docs/heat_units.md]: Can we use subscripts for math notation?

```suggestion
  $$ T_{\text{min}} = 
  max(T_{\text{air, min}}, T_{\text{crop, min}})     
  $$
```

File: SC_redesign/Crop_and_Soil/docs/heat_units.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```
+where *Tmin* is `minimum_heat_unit_temperature`, *TminAir* is `minimum_air_temperature`, the minimum temperature on 
+a given day, and *TminCrop* is `minimum_growth_temperature`, base or minimum temperature or the crop to growth.
```
Comment (morrowcj) [inline SC_redesign/Crop_and_Soil/docs/heat_units.md]: And to make sure that the symbols in the equations match the symbols in your text, you can surround terms with `$` in the markdown:

```suggestion
where $T_{\text{min}}$ is `minimum_heat_unit_temperature`, T_{\text{air, min}} is `minimum_air_temperature`, the minimum temperature on 
a given day, and T_{\text{crop, min}} is `minimum_growth_temperature`, base or minimum temperature or the crop to growth.

```

**Review by morrowcj — APPROVED** [review]
Looks good to me. I just had a small note about notation in math mode.

File: SC_redesign/Crop_and_Soil/docs/heat_units.md (hunk 1 above)
Comment (LaplacetteCM) [reply SC_redesign/Crop_and_Soil/docs/heat_units.md]: I didn't find anything about not using them but I've sent Pooya a message to double check

File: SC_redesign/Crop_and_Soil/docs/heat_units.md (hunk 2 above)
Comment (LaplacetteCM) [reply SC_redesign/Crop_and_Soil/docs/heat_units.md]: Nice, thanks!

**Review by morrowcj — APPROVED** [review]
looks good to me.

**Pull request merged**